fleche/config.ml

```diff
@@ -37,6 +37,8 @@ type t =
         (** Pretty-printing type in Info Panel Request, 0 = string; 1 = Pp.t; 2
             = Coq Layout Engine *)
   ; show_stats_on_hover : bool [@default false]
+  ; verbosity : int [@default 2]
```

Comment:
@HazardousPeach do you recall what the values do mean, I'd like to document them.